products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -205,57 +206,77 @@ public struct ProcessHandle: Hashable {
     public var hashValue: Int {
         return handle.hashValue
     }
-}
-public func ==(lhs: ProcessHandle, rhs: ProcessHandle) -> Bool {
-    return lhs.handle == rhs.handle
+
+    public static func ==(lhs: ProcessHandle, rhs: ProcessHandle) -> Bool {
+        return lhs.handle == rhs.handle
+    }
 }
 
 /// Result of a command execution.
 public enum CommandResult {
-  case succeeded
-  case failed
-  case cancelled
-  case skipped
-
-  init(_ result: llb_buildsystem_command_result_t) {
-    switch result {
-    case llb_buildsystem_command_result_succeeded:
-      self = .succeeded
-    case llb_buildsystem_command_result_failed:
-      self = .failed
-    case llb_buildsystem_command_result_cancelled:
-      self = .cancelled
-    case llb_buildsystem_command_result_skipped:
-      self = .skipped
-    default:
-      fatalError("unknown command result")
-    }
-  }
+    case succeeded
+    case failed
+    case cancelled
+    case skipped
+
+    init(_ result: llb_buildsystem_command_result_t) {
+        switch result {
+        case llb_buildsystem_command_result_succeeded:
+            self = .succeeded
+        case llb_buildsystem_command_result_failed:
+            self = .failed
+        case llb_buildsystem_command_result_cancelled:
+            self = .cancelled
+        case llb_buildsystem_command_result_skipped:
+            self = .skipped
+        default:
+            fatalError("unknown command result")
+        }
+    }
+}
+
+public struct CommandMetrics {
+    public let utime: UInt64         /// User time (in us)
```

Comment:
These comments (originally written by me) are unfortunately incorrect. We are currently returning the times in nanoseconds.